logo.svg

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" standalone="no"?>
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
+<svg xmlns="http://www.w3.org/2000/svg" version="1.1" viewBox="0,0 7,7" width="196px" height="196px">
+  <path fill="#f60" d="M 0,0 L 2,0 L 0,2 L 0,0" />
+  <path fill="#f90" d="M 2,0 L 4,0 L 0,4 L 0,2 L 2,0" />
+  <path fill="#fc0" d="M 4,0 L 6,0 L 0,6 L 0,4 L 4,0" />
+  <path fill="#9e0" d="M 6,0 L 7,0 L 7,1 L 1,7 L 0,7 L 0,6 L 6,0" />
+  <path fill="#6cf" d="M 7,1 L 7,3 L 3,7 L 1,7 L 7,1" />
+  <path fill="#66f" d="M 7,3 L 7,5 L 5,7 L 3,7 L 7,3" />
+  <path fill="#60f" d="M 7,5 L 7,7 L 5,7 L 7,5" />
+  <path fill="#fff" d="M 1,1 L 3,1 L 3,2 L 2,2 L 2,3 L 3,3 L 3,4 L 2,4 L 2,6 L 1,6 L 1,1 M 4,1 L 5,1 L 5,5 L 6,5 L 6,6 L 4,6 L 4,1" />
+</svg>
```

Comment:
![Fantasy Land logo](https://cdn.jsdelivr.net/gh/fantasyland/fantasy-land@350ab781cf46b69b998ada2a9c9c812f35cd6e75/logo.svg)

---

I've been thinking of proposing a new Fantasy Land logo for some time. If we are to release a dramatically different version of the specification, I propose that it be accompanied by a new logo (the logo above is one option). We could encourage project maintainers to adopt the new logo coincident with supporting the new version of the specification, so users could tell at a glance which projects support the new version.

@puffnfresh, did you create the current logo? How do you feel about the possibility of replacing it?